tests/contrib/wsgi/test_wsgi.py

```diff
@@ -30,11 +30,25 @@ def chunked_response_generator_error(start_response):
             raise GeneratorExit()
 
 
+def raise_stop_iteration_exception(start_response):
+    status = "200 OK"
+    headers = [("Content-type", "text/plain")]
+    start_response(status, headers)
+
+    def g():
+        yield 13
+
+    it = g()
+    next(it), next(it)
```

Comment:
nit: This endpoint feels a bit off. Can we just raise StopIteration here instead of creating a generator and calling next twice. Some thing like this could also work `next(list().__iter__())`

Reply:
https://github.com/DataDog/dd-trace-py/pull/8156#discussion_r1463510049 Sure, I think they're basically the same but happy to make it simpler.